docs/source/_src_user_guide/state_propagation/propagation_setup/dynamics_types/multi_arc.rst

```diff
@@ -0,0 +1,33 @@
+.. _multi_arc_dynamics:
+
+==============================
+Multi- and hybrid-arc dynamics
+==============================
+
+Multi-arc dynamics
+------------------
+
+For various types of applications, the dynamics of a given body will not be propagated over a single continuous arc, but will be split over several independent propagations that may or may not be overlapping in time, or contiguous in state. A number of examples are:
+
+* Orbit determination of a spacecraft, where the state is typically propagated (and estimated) over a set of (partially overlapping) period of several days, to prevent dynamical model errors from accumulating in the state estimation/
+* Propagating spacecraft dynamics only during specific periods of interest. For instance, for a spacecraft like JUICE or Cassini, the user may only be interested in propagating the dynamics of the spacecraft during the flybys of the Jovian/Saturnian moons, without needing a numerical result in the intermediate time periods.
+
+Multi-arc propagation settings are defined using the :func:`~tudatpy.numerical_simulation.propagation_setup.propagator.multi_arc` function. For the multi-arc propagation, the primary input is a list of single-arc settings (see :ref:`here <propagation_setup_intro>` for a list of single-arc dynamics types), which define the dynamics type, initial time, initial state, etc. to be used for the separate arcs. The propagations overe the separate single arcs are done almost entirely independently, with a number of (optional) links between the different arcs:
+
+* The processing of the numerical results (most specificall, resetting the ephemeris of the body or not, see :ref:`auto_processing`) is defined consistently for each of the arcs in the multi-arc propagation. NOTE : the processing settings in the single-arc propagations are automatically overwritten by the multi-arc processing settings.
```

Comment:
'most specificall', I don't know exactly what you mean. Maybe 'in particular', or 'for example', or 'specifically'?

Reply:
in particular would be best

Reply:
Fixed